server/middlewares/proxy.ts

```diff
@@ -0,0 +1,19 @@
+const envs = require("env-smart")
```

Comment:
You don't need `env-smart`, just use `process.env.HTTP_PROXY` and `process.env.HTTPS_PROXY`

Reply:
We have used the process.env.HTTP_PROXY & process.env.HTTPS_PROXY with this changes.